samples/local_attestation/enclave_b/Makefile

```diff
@@ -36,7 +36,7 @@ build:
 	oeedger8r ../localattestation.edl --trusted --trusted-dir ../common \
 		--search-path $(INCDIR) \
 		--search-path $(INCDIR)/openenclave/edl/sgx
-	$(CXX) -g -c $(CXXFLAGS) $(INCLUDES) -I. -I.. -std=c++11 -DOE_API_VERSION=2 ecalls.cpp ../common/attestation.cpp ../common/crypto.cpp ../common/dispatcher.cpp
+	$(CXX) -g -c $(CXXFLAGS) $(INCLUDES) -I../../../include -I. -I.. -std=c++11 -DOE_API_VERSION=2 ecalls.cpp ../common/attestation.cpp ../common/crypto.cpp ../common/dispatcher.cpp
```

Comment:
Why is it needed? /opt/openenclave/include should already be in include path since you are required to run 'source /opt/openenclave/share/openenclave/openenclavec' before building samples.

Reply:
This is a temporary fix for until #3309 is resolved.

Reply:
Got it.